test/spec/unit/core/storageManager_spec.js

```diff
@@ -42,5 +42,34 @@ describe('storage manager', function() {
     storage.setCookie('foo1', 'baz1');
     expect(deviceAccessSpy.calledOnce).to.equal(true);
     deviceAccessSpy.restore();
+  });
+
+  describe('localstorage forbidden access in 3rd-party context', function() {
+    let errorLogSpy;
+
+    beforeEach(function() {
+      var mock = {
+        get: function() {
+          throw Error;
+        }
+      };
+      Object.defineProperty(window, 'localStorage', mock);
```

Comment:
Hi @Swiiip 

Can you remove this property as part of the `afterEach()`?  From how it looks now, this mocked function may be available for other tests in other files (since we generally run everything together).

Thanks

Reply:
Done, I reset the property descriptor after each test